flight/Modules/System/systemmod.c

```diff
@@ -44,7 +44,9 @@
 #include "watchdogstatus.h"
```

Comment:
Whats the rationale for moving all the periodic callback stuff from the uavo manager to systemmod? To me, it seems more logical to have it in the uavo manager.